index.js

```diff
@@ -143,8 +143,15 @@ HtmlWebpackPlugin.prototype.apply = function (compiler) {
           });
       })
       .then(function (html) {
-        // Add the stylesheets, scripts and so on to the resulting html
-        return self.postProcessHtml(html, assets);
+        // Prepare script and link tags
+        var assetTags = self.generateAssetTags(assets);
+        var pluginArgs = {head: assetTags.head, body: assetTags.body, plugin: self, outputName: self.childCompilationOutputName};
```

Comment:
Ah, thanks. Doing that. 

-- 
Pierce Wetter
Sent with Airmail

From: Jan Nicklas <notifications@github.com>
Reply: ampedandwired/html-webpack-plugin <reply@reply.github.com>
Date: June 9, 2016 at 10:37:51 PM
To: ampedandwired/html-webpack-plugin <html-webpack-plugin@noreply.github.com>
Cc: SPSpwetter <pwetter@skyportsystems.com>, Comment <comment@noreply.github.com>
Subject:  Re: [ampedandwired/html-webpack-plugin] Add 'html-webpack-plugin-alter-asset-tags' (#345)  

In index.js:

> @@ -143,8 +143,15 @@ HtmlWebpackPlugin.prototype.apply = function (compiler) {
>            });
>        })
>        .then(function (html) {
> -        // Add the stylesheets, scripts and so on to the resulting html
> -        return self.postProcessHtml(html, assets);
> +        // Prepare script and link tags
> +        var assetTags = self.generateAssetTags(assets);
> +        var pluginArgs = {head: assetTags.head, body: assetTags.body, plugin: self, outputName: self.childCompilationOutputName};
Well that is already available in the plugin context - so you could do sth like:

compiler.plugin('compilation', function(compilation) {
   compilation.plugin('html-webpack-plugin-alter-asset-tags', callback.bind(this, compilation));
}).bind(this);

function callback(compilation, htmlWebpackPluginData) {

}
—
You are receiving this because you commented.
Reply to this email directly, view it on GitHub, or mute the thread.